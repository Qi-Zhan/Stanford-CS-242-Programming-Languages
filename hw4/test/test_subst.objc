START VERIFY;

x = [l1 = \o.o, l2 = \o.o];
y = [l1 = \o.x, l2 = \o.x];
z = y.l1;

STOP VERIFY;
