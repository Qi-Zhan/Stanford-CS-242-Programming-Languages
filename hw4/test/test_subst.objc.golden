x = [l1 = \o.(o), l2 = \o.(o)];
y = [l1 = \o.([l1 = \o.(o), l2 = \o.(o)]), l2 = \o.([l1 = \o.(o), l2 = \o.(o)])];
z = [l1 = \o_0.(o_0), l2 = \o_0.(o_0)];
